src/main/kotlin/uk/gov/gdx/datashare/services/SuppliersService.kt

```diff
@@ -107,6 +107,7 @@ class SuppliersService(
     )
     val otherSubscriptionsWithClient = supplierSubscriptionRepository.findAllByClientId(subscription.clientId)
     if (otherSubscriptionsWithClient.isEmpty()) {
+      AcquirersService.log.info("Deleting User Pool Client ID: ${subscription.clientId}")
```

Comment:
Should this be using the `AcquirersService` logger?

Reply:
I had hoped it would be smarter than this :(